fluent-bit/map.jinja

```diff
@@ -20,7 +20,7 @@ that differ from whats in defaults.yaml
 {% set init_map = salt['grains.filter_by']({
     'systemd': {
       "service": {
-        "conf": '/lib/systemd/system/fluent-bit.service',
+        "conf": '/etc/init/fluent-bit.conf',
```

Comment:
Why this change? This is not `systemd` conf.

Reply:
Hm, this looks like as mistake on my side, I need to check that. Thanks for catch